tools/ARIAtools/ARIAProduct.py

```diff
@@ -433,12 +433,21 @@ def __mappingVersion__(self, fname, version):
                 '/science/grids/imagingGeometry/azimuthAngle'
             ]
             if version.lower()=='1c':
+                # get weather model name
```

Comment:
line 406. How is scene lenght used again? What are implications of use adding more overlap or making frames larger?